call-a-function-api-oci-clouddev.md

```diff
@@ -0,0 +1,772 @@
+---
+layout: collection
```

Comment:
Still including keys that should NOT be in a post. This is not a collection, and `sort` has no effect on an individual post. Please ensure you're copy/pasting the correct headers.